drivers/flash/nrf_qspi_nor.c

```diff
@@ -786,6 +801,16 @@ static int qspi_nrfx_configure(const struct device *dev)
 		}
 	}
 
+#ifdef CONFIG_PM_DEVICE_RUNTIME
+	ret = pm_device_runtime_enable(dev);
+	if (ret < 0) {
+		LOG_ERR("Failed to enable runtime power management: %d", ret);
+	} else {
+		LOG_DBG("Runtime power management enabled");
+	}
+#endif
```

Comment:
Why not do this in `qspi_nor_configure()` where `qspi_device_uninit()` is called (both paths would then initially suspend the device in the same place)?

Reply:
Good catch, that makes the logic more clear, I'll update that